packages/react-server/src/ReactFlightServer.js

```diff
@@ -533,6 +534,10 @@ function serializeProviderReference(name: string): string {
   return '$P' + name;
 }
 
+function serializeUndefined(): string {
+  return '$U';
```

Comment:
I have a scheme in my notes for all the missing ones. For undefined I had written down “$undefined”. The parsing can just check for “u”. It doesn’t conflict with anything. But the full word might make it a little easier to debug when you’re manually reading it. It’s wasting some bytes but so does a lot of thing.

A similar one is Date. Was thinking about whether it should use the string or number form.